### PR TITLE
Added basic static footer component

### PR DESCRIPTION
**Pull request by bartaz**
Simple footer. Design based on SnapWeb and SCA for consistency:

my-ubuntu-footer-static

### CODE REVIEW

File: src/style/base.css [hunk 1]
```diff
@@ -56,6 +56,11 @@ a:focus {
   outline: thin dotted;
 }
 
+html {
+  min-height: 100%;
+  position: relative;
```
Comment (bartaz) [inline src/style/base.css]: Needed to keep absolute positioned footer at the bottom.

File: src/components/footer/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+import styles from './footer.css';
+
+export default class Footer extends Component {
+  render() {
+    return (
+      <div className={ styles.footer }>
+        <div className={ styles.container }>
+          <p className={ styles.copyright }>© 2016 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.</p>
+          <p><a href="#">Terms of Service</a> · <a href="https://github.com/canonical-ols/javan-rhino/issues/new">Report a bug on this site</a></p>
```
Comment (bartaz) [inline src/components/footer/index.js]: We need to define what links we want in footer and where should they point to.

Comment (earnubs) [inline src/components/footer/index.js]: I'd expect the TOS to be on my.ubuntu.com, for content we'll need to raise that with @evandandrea 

Report a bug can point to github issues for now.

Comment (bartaz) [inline src/components/footer/index.js]: Added a card to Trello about that.

**Pull request merged**